### PR TITLE
Fixed amchart in small screen

### PR DESCRIPTION
**Pull request by amitsin6h**
Added horizontal and vertical scroll bar in iframe for small devices

### CODE REVIEW

**Comment by amitsin6h** [comment]
@kracekumar @realslimshanky

**Comment by realslimshanky** [comment]
We still have a lot of space left to the right of the map. Try increasing the width.
Keep the height same as the size of the image.
image

**Comment by amitsin6h** [comment]
@realslimshanky height is same. I have only increased the width. Hope it works now

File: themes/inpycon2017/static/js/jquery.responsiveiframe.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    return new ResponsiveIframe();
+  }
+
+  // expose
+  if ('undefined' === typeof exports) {
+    window.responsiveIframe = responsiveIframe;
+  } else {
+    module.exports.responsiveIframe = responsiveIframe;
+  }
+}());
```
Comment (realslimshanky) [inline themes/inpycon2017/static/js/jquery.responsiveiframe.js]: Please add a newline here.

**Comment by realslimshanky** [comment]
@amitsin6h remove the constraint of height (i.e. max-height etc) and make the max width 450px since there is a lot of space left on the right hand side. Also, make sure all these properties work on the iframe only for a mobile display. Use media query for that.

File: themes/inpycon2017/static/js/jquery.responsiveiframe.js (hunk 1 above)
Comment (amitsin6h) [reply themes/inpycon2017/static/js/jquery.responsiveiframe.js]: @realslimshanky I have removed this lib. This does not works

**Comment by amitsin6h** [comment]
@realslimshanky I have checked it works well in mobile device.

**Comment by realslimshanky** [comment]
@amitsin6h thank for your contribution :pizza:

**Pull request merged**